main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: all
+  connection: local
+
+  vars_files:
+    - default.config.yml
+
+  roles:
+    - role: uninstalls
+      tags: ['uninstalls']
+    - role: command-line
+      tags: ['command-line']
+    - role: homebrew
+      tags: ['homebrew']
+    - role: rbenv
+      tags: ['rbenv']
+    - role: gems
+      tags: ['gems']
+    - role: postgresql
+      tags: ['postgresql']
+    - role: phpunit
+      tags: ['phpunit']
+    - role: wp-cli
+      tags: ['wp-cli']
+    - role: github
+      tags: ['github']
+    - role: signage-setup
+      tags: ['signage']
+    - role: blogs-setup
+      tags: ['blogs']
```

Comment:
Looking at the blogs role, I think these app roles (blogs-setup, inside-setup, signage-setup) should not be included in this playbook. I prefer to treat them as a separate concern.

Leave them in the repo, but comment them out in this file for now.